packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -96,6 +96,17 @@ export function toRemovalInfo(
 	);
 }
 
+/**
+ * Tracks information about each concurrent move operation which affected a segment.
+ * @legacy
+ * @alpha
+ */
+export interface IConcurrentMoveInfo {
+	clientId: number;
```

Comment:
Do these properties get mutated? Or can we make them `readonly`?

Reply:
Also, we should add docs to these properties if we are exporting the parent type.

Reply:
The properties do get mutated, so I don't think they should be `readonly`. I'll add docs though.